examples/pytorch/pointcloud/pointnet/README.md

```diff
@@ -7,13 +7,21 @@ This is a reproduction of the papers
 
 # Performance
 
+## Classification
```

Comment:
Add numbers from the original paper and the reference implementation to these two tables

Reply:
Added numbers from the original paper and from the other GitHub repo

Reply:
Where are the numbers?

Reply:
Fixed